### PR TITLE
Feature sb 74 product preview page

### PR DESCRIPTION
**Pull request by ValentinaTenus**
### **General**
- [x] Assigned myself to the PR
- [x] Assigned the appropriate labels to the PR
- [x] Assigned the appropriate reviewers to the PR
- [ ] Updated the documentation
- [x] Performed a self-review of my code
- [x] Types for input and output parameters
- [x] Don't have "any" on my code
- [x] Used the try/catch pattern for error handling
- [x] Don't have magic numbers
- [x] Compare only with constants not with strings
- [x] No ternary operator inside the ternary operator
- [x] Don't have commented code
- [x] no links in the code, env links should be in env file
- [x] Used camelCase for variables and functions
- [x] Date and time formats are on the constants
- [x] Functions are public only if it's used outside the class
- [x] No hardcoded values
- [ ] covered by tests
- [x] Check your commit messages meet
### Frontend
- [x] Components and business logic are separated
- [x] Colors, Font Size, and Font Name is on the theme or in the
- [x] No text in the components, use i18n approach
- [x] No inline styles
- [x] Imports are absolute
- [x] Attach a screenshot if PR has visual changes.

1. changed styles on product page to make it unclickable and unchangebale for vendor in preview mode.
2. add new modal window which apears when vendor click anywhere on page

image
image
image

### CODE REVIEW

File: src/helpers/sort-sizes.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const sortOrder = ["XS", "S", "M", "L", "XL"];
```
Comment (VladManukian) [inline src/helpers/sort-sizes.ts]: We need more sort option
[Size](https://github.com/ZenBit-Tech/sunny-bunnies_fe/blob/develop/src/pages/profile-board/constants.ts/size.ts)

File: src/pages/product/components/vendor-preview-mode-modal/index.tsx [hunk 2]
```diff
@@ -30,22 +31,35 @@ const VendorPreviewModeModal: React.FC<VendorPreviewModeModalProperties> = ({
...
+					</StyledVendorPreviewIcon>
+					<Typography
+						sx={{ fontSize: fontSizes.extraLarge, textAlign: "center" }}
+						variant="playfairDisplayBold"
+					>
+						{t("ProductPage.youAreInPreviewMode")}
+					</Typography>
 					<Typography
-						sx={{ fontSize: fontSizes.mediumLarge, lineHeight: "30px" }}
+						sx={{ color: colors.secondaryTextGray, textAlign: "center" }}
```
Comment (VladManukian) [inline src/pages/product/components/vendor-preview-mode-modal/index.tsx]: In my opinion, we should use theme for all colors, font size, font weight from now.

File: src/pages/product/components/product-buttons-group/index.tsx [hunk 3]
```diff
@@ -28,22 +28,18 @@ const ProductButtonsGroup: React.FC<ProductButtonsGroupProperties> = ({
...
-					{t("ProductPage.makeAnOffer")}
-				</BaseButton>
-			)}
+			</StyledBaseButton>
 			<Box display="flex" gap="32px">
 				<BaseButton
 					onClick={handleLikeButtonClick}
 					startIcon={isLikeClicked ? <FavoriteIcon /> : <FavoriteBorderIcon />}
-					sx={{ fill: "black", textTransform: "none" }}
+					sx={{ fill: "black", pointerEvents: "none", textTransform: "none" }}
```
Comment (VladManukian) [inline src/pages/product/components/product-buttons-group/index.tsx]:
```suggestion
					sx={{ fill: theme.palette,primary.black pointerEvents: "none", textTransform: "none" }}
```
or add custom black and use it

File: src/libs/constants/fonts.ts [hunk 4]
```diff
@@ -1,4 +1,5 @@
 const fontSizes = {
+	extraLarge: "36px",
```
Comment (VladManukian) [inline src/libs/constants/fonts.ts:2]: update theme -> fontSize too

File: src/helpers/sort-sizes.ts [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+	"3XS",
+	"XXS",
+	"XS",
+	"S",
+	"M",
+	"L",
+	"XL",
+	"XXL",
+	"XXXL",
+];
```
Comment (lyudad) [inline src/helpers/sort-sizes.ts]: move to constants (folder and file). no one will find it here

File: src/libs/constants/fonts.ts (hunk 4 above)
Comment (lyudad) [reply src/libs/constants/fonts.ts:2]: dont see update

Comment (ValentinaTenus) [reply src/libs/constants/fonts.ts:2]: we use  these constants with colors, fonts sizes and weights to define our colors and fonts in theme. I've added it to theme and in code use only from theme.

File: src/helpers/sort-sizes.ts (hunk 5 above)
Comment (ValentinaTenus) [reply src/helpers/sort-sizes.ts]: I've moved const sortOrder to constants, but left function and folder. we have already another files with helpers functions in this folder. it is our structure 
image
if it's not good I'll change

**Review by lyudad — APPROVED** [review]

**Pull request merged**